keyboards/matrix/pjf/rules.mk

```diff
@@ -0,0 +1,20 @@
+# MCU
+MCU  = STM32F103
+BOARD = pjf_keyboard
+MCU_LDSCRIPT = pjf
+
+# options
+BOOTMAGIC_ENABLE = yes # Virtual DIP switch configuration
+## (Note that for BOOTMAGIC on Teensy LC you have to use a custom .ld script.)
+MOUSEKEY_ENABLE = no # Mouse keys
+EXTRAKEY_ENABLE = yes # Audio control and System control
+CONSOLE_ENABLE = no# Console for debug
+COMMAND_ENABLE = no # Commands for debug and configuration
+NKRO_ENABLE = no # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+
+CUSTOM_MATRIX = yes
+RGBLIGHT_CUSTOM_DRIVER = yes
+
+SRC += drivers/arm/ws2812.c matrix.c
```

Comment:
Is the custom driver still needed?

Reply:
yes, this keyboard matrix row was pull down by an external resistor. the current matrix codes can't handle such cases.